Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -68,6 +68,23 @@ ErrorOr<void> PresenterWidget::initialize_menubar()
         this->window()->set_fullscreen(true);
     })));
 
+    auto black_background_action = GUI::Action::create("&Black background", { KeyCode::Key_B }, [this](auto&) {
+        if (m_current_presentation) {
+            m_current_presentation->set_color_scheme(Color::Black, Color::LightGray);
+            update();
+        }
+    });
+    auto white_background_action = GUI::Action::create("&White background", { KeyCode::Key_W }, [this](auto&) {
+        if (m_current_presentation) {
+            m_current_presentation->set_color_scheme(Color::White, Color::DarkGray);
+            update();
+        }
+    });
+    TRY(presentation_menu.try_add_action(black_background_action));
+    TRY(presentation_menu.try_add_action(white_background_action));
+    m_black_background_action = black_background_action;
+    m_white_background_action = white_background_action;
```

Comment:
This belongs in a separate commit; it adds the black-out and white-out functionality which relates to color scheme but is not the same (it's just mentioned in the documentation alongside).